### PR TITLE
Python session 01

### PR DESCRIPTION
**Pull request by amin-rami**
Python session-01

### CODE REVIEW

**Review by garshasbi-f — DISMISSED** [review]

File: python/session-01.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+        Outside opinion
+        Outside opinion
+```
+![Mindomo](https://www.mindomo.com/c/wp-content/uploads/2022/05/3.-Problem-Solving.png)
+You can download Mindomo for free from it's official website
+
+---
+## Obsidian
+Obsidian is simply an interface for plain text files that are stored on your computer, and supports standard Markdown formatting. Many themes will have syntax highlighting to make it easier to see your Markdown-formatted text, but the Markdown that Obsidian supports will be viewable in any other text editor.
+
```
Comment (Tahoora78) [inline python/session-01.md:175]: Obsidian is a powerful knowledge base that works on top of a local folder of plain text Markdown files. In Obsidian, making and following connections is frictionless, and with the connections in place, you can explore all of your knowledge in the interactive graph view. Obsidian supports CommonMark and GitHub Flavored Markdown (GFM), along with other useful notetaking features such as tags, LaTeX mathematical expressions, mermaid diagrams, footnotes, internal links and embedding Obsidian notes or external files. Obsidian also has a plugin system with 20+ supported plugins to expand its capabilities. The plugin system is currently a work-in-progress, there will be public APIs available to developers in the v1.0 release.
for more information you can visit this <a href="https://obsidian.md/#:~:text=Obsidian%20is%20a%20powerful%20knowledge,%2C%20and%20Linux%20(Flatpak).">link</a>

Comment (amin-rami) [reply python/session-01.md:175]: Thanks for the feedback

**Review by M-Heidari2000 — DISMISSED** [review]

**Review by M-Heidari2000 — APPROVED** [review]

**Review by Tahoora78 — APPROVED** [review]
approved

File: python/session-01.md (hunk 1 above)
Comment (Tahoora78) [reply python/session-01.md:175]: your welcome

**Review by fmahmoudii — APPROVED** [review]

**Pull request merged**